docs/self-managed/zeebe-deployment/operations/backups.md

```diff
@@ -0,0 +1,81 @@
+---
+id: backups
+title: "Backups"
+description: "Overview and Guide for taking Zeebe backups"
+---
+
+Zeebe fully manages the state of your process instances and does not use an external DBMS, you might want to consider taking backups of Zeebe data.
+Zeebe is fault tolerant and replicates state internally.
+Backups are only necessary if you want to protect against the loss of entire replica sets or data corruption bugs.
+
+State of other component, such as Operate and Tasklist, is not managed by Zeebe and needs to be backed up separately.
+
+Taking backups is a manual process that is highly dependent on you infrastructure and deployment.
+We do not provide an automated backup mechanism or tool.
+
+## Cold Backups
+
+Cold backups, also called offline backups, require **downtime**.
+During the downtime, processes don't make progress and clients can't communicate with Zeebe.
+We recommend that you test how your clients behave during the downtime or that you shut down clients as well.
+
+### Shutting down all brokers in you cluster
+To take a consistent backup, all brokers must be shut down first.
+
+As soon as brokers are shutting down, partitions become unhealthy and clients will lose connections to Zeebe or experience full backpressure.
+To prevent unnecessary failovers during the shutdown process, we recommend to shut down all brokers at the same time instead of a gradual shutdown.
+
+### Creating the backup
+:::caution
+The `data` folder might contain symbolic and hard links which require special attention.
+:::
+
+Each broker has a data folder where all state is persisted.
+The location of the data folder is [configured](../configuration/configuration.md) via `zeebe.broker.data.directory`.
+To save the the data folder you can either create an archive of the entire folder or take filesystem or volume snapshots.
+
+Do not merge or otherwise modify data folders. Store the backup such that each data folder is associable to their broker.
```

Comment:
The second sentence is hard to read. "Store the backup such that each data folder is associable to their broker." I do not fully get where I should store my backup now. 


I assume that this is the normal location where my application would access it?

Reply:
Yeah, I agree that's not great writing :D I'll rephrase to make it a bit more clear :+1:

Reply:
I'll just remove this sentence. What I wanted to say was that no one should start merging all the data folders from the brokers. I think this is clear enough with the example I give below.